crates/sui-core/src/node_sync/node_state.rs

```diff
@@ -63,7 +63,7 @@ macro_rules! check_epoch {
 
         if expected_epoch != observed_epoch {
             // Most likely indicates a reconfiguration bug.
-            error!(?expected_epoch, ?observed_epoch, "Epoch mis-match");
+            error!(?expected_epoch, ?observed_epoch, "Epoch miss-match");
```

Comment:
I think "mismatch" with one s and no hyphen is the most correct spelling

Reply:
lol `typo` failure.